pisa/utils/fileio.py

```diff
@@ -401,7 +401,15 @@ def from_cfg(fname):
 def from_pickle(fname):
     """Load from a Python pickle file"""
     try:
-        return pickle.load(file(fname, 'rb'))
+        # Open the file (binary)
+        f = open(fname, 'rb')
+        try :
+            # Try standard pickle load
+            return pickle.load(f)
+        except :
```

Comment:
can you add the specific exception? Such that only the one with the wrong encoding is handled
If there is another exception, like missing file, it should be thrown

Reply:
done